deployment/cdk/opensearch-service-migration/cdk.context.json

```diff
@@ -1,17 +1,17 @@
 {
   "demo-deploy": {
     "stage": "dev",
-    "engineVersion": "OS_2.9",
-    "domainName": "demo-opensearch-cluster",
-    "dataNodeCount": 2,
-    "openAccessPolicyEnabled": true,
-    "domainRemovalPolicy": "DESTROY",
-    "artifactBucketRemovalPolicy": "DESTROY",
-    "enableDemoAdmin": true,
-    "trafficReplayerServiceEnabled": true,
-    "trafficReplayerEnableClusterFGACAuth": true,
-    "captureProxyESServiceEnabled": true,
+    "targetCluster": {
+      "endpoint": "<TARGET_CLUSTER_ENDPOINT>",
+      "auth": { "type": "none" }
+    },
+    "sourceCluster": {
+      "endpoint": "<SOURCE_CLUSTER_ENDPOINT>",
+      "auth": { "type": "none" }
+    },
+    "vpcId": "<VPC_ID>",
     "reindexFromSnapshotServiceEnabled": true,
+    "trafficReplayerServiceEnabled": true,
     "otelCollectorEnabled": true
```

Comment:
I don't think this should be changed right now but this is an implementation detail. A user shouldn't have to know what OTEL is to use our solution.

Reply:
Oh, yeah, I actually meant to pull this one out. It's set to true in the defaults, so they get it anyways